File: SAST/javascript/sample.js
function insecureFunction(password) {
    console.log("Received password: " + password);
  }
  
  var userInput = "sensitivePassword";
  insecureFunction(userInput);

// (CWE-359)
// This sample JavaScript file contains an insecure function that logs a password to the console. 
// It can be used to test SAST tools' ability to identify sensitive information exposure.

File: SAST/python/sample.py
import os

def insecure_function(password):
    print("Received password: " + password)

user_input = "sensitivePassword"
insecure_function(user_input)

# (CWE-359)
# This sample Python file contains a function that prints a password to the console without any security measures.
# It can be used to test SAST tools' ability to identify sensitive information exposure.